docs/src/main/sphinx/release/release-409.md

```diff
@@ -0,0 +1,72 @@
+# Release 409 (1 Mar 2023)
+
+## General
+
+* Add support for dropping fields with a `DROP COLUMN` statement. ({issue}`15975`)
+* Remove the `distributed-index-joins-enabled` configuration property and
+  related functionality. ({issue}`15375`)
+* Fix failure when using non-comparable and non-sortable fields as part of a
+  `VALUES` expression. ({issue}`)
+
+## BigQuery connector
+
+* Fix failure when non-lowercase column names exist in the `query` table
+  function. ({issue}`16075`)
+
+## ClickHouse connector
+
+* Remove support for the `ru.yandex.clickhouse.ClickHouseDriver` legacy JDBC
+  driver. ({issue}`16188`)
+* Remove support for specifying expressions in the `sample_by` table property to
+  prevent SQL injection. ({issue}`16261`)
+
+## Delta Lake connector
+
+* Add support for setting default reader and writer versions using the
```

Comment:
what writer are we talking about here .. ORC, Parquet, Delta Lake, or something else .. also .. is there a link or info on valid values in the docs?

Reply:
Please remove this entry. Relates to #16310

Reply:
BTW seems "Add support for setting default reader and writer versions" was part of 408 (https://github.com/trinodb/trino/pull/16332). and we backed it out, replacing with an automatic approach now https://github.com/trinodb/trino/pull/16310